game/lib/gameplay/gameplay_page.dart

```diff
@@ -2,9 +2,11 @@ import 'package:flutter/material.dart';
 import 'package:flutter/src/foundation/key.dart';
 import 'package:flutter/src/widgets/framework.dart';
 import 'package:flutter/src/widgets/placeholder.dart';
+import 'package:velocity_x/velocity_x.dart';
 
 import 'gameplay_map.dart';
 
+// Returns widget that fills based on tasks and tasks_completed.
 List<Widget> generate_progress(tasks, tasks_completed) {
   List<Widget> progress_blocks = [];
   for (var i = 0; i < tasks; i++) {
```

Comment:
Use of tasks here is confusing, an explicit type would be helpful. Consider changing to something like taskCount.